pylutron/__init__.py

```diff
@@ -561,6 +562,11 @@ def add_button(self, button):
     dispatch button events."""
     self._buttons.append(button)
 
+  def press(self, button):
```

Comment:
I wonder if this should be triggered via the button object itself? The fact that you are reaching into the button private for _num seems to tell me that we should have a cleaner interface. I never fully flushed out the keypad <> button relationship as I was just trying to parse most things.

Though, if we do that, we'll need to provide our integration id to the button, or maybe add something like button->set_keypad(...) or set_owner, something like that.

Then we'd just be able to button->trigger_press() or maybe button->toggle() or something like that.

Reply:
We can also change the parser to just hand the button our keypad object at button creation.